src/main/java/uk/org/openbanking/datamodel/fund/OBErrorResponse1.java

```diff
@@ -0,0 +1,182 @@
+/**
+ *
+ * The contents of this file are subject to the terms of the Common Development and
+ *  Distribution License (the License). You may not use this file except in compliance with the
+ *  License.
+ *
+ *  You can obtain a copy of the License at https://forgerock.org/cddlv1-0/. See the License for the
+ *  specific language governing permission and limitations under the License.
+ *
+ *  When distributing Covered Software, include this CDDL Header Notice in each file and include
+ *  the License file at legal/CDDLv1.0.txt. If applicable, add the following below the CDDL
+ *  Header, with the fields enclosed by brackets [] replaced by your own identifying
+ *  information: "Portions copyright [year] [name of copyright owner]".
+ *
+ *  Copyright 2018 ForgeRock AS.
+ */
+/*
+ * Confirmation of Funds API Specification
+ * Swagger for Confirmation of Funds API Specification
+ *
+ * OpenAPI spec version: v3.0.0
+ * Contact: ServiceDesk@openbanking.org.uk
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+
+package uk.org.openbanking.datamodel.fund;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * An array of detail error codes, and messages, and URLs to documentation to help remediation.
+ */
+@ApiModel(description = "An array of detail error codes, and messages, and URLs to documentation to help remediation.")
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaClientCodegen", date = "2018-11-15T11:58:26.525Z")
+public class OBErrorResponse1 {
```

Comment:
Think it exists as https://github.com/ForgeRock/OpenBanking-Java-SDK/blob/master/src/main/java/uk/org/openbanking/datamodel/error/OBErrorResponse1.java